### ISSUE TITLE
Documentation -- meaning of detection time

### ISSUE THREAD

**Issue by tctbeg** [eqcorrscan/EQcorrscan#474]
This issue is just to request/remind about documenting the meaning of detection time. How are the detection times calculated? As the earliest time in the template + the shift required to move the template to the detection?

**Comment by flixha** [comment]
Yes, the detection time is set according to the earliest time in the template (i.e. the starttime of the earliest trace in the template) plus the shift required to move the template to the detection.

The starttime of the earliest trace in the template is fixed during template creation, it uses the earliest pick for which waveform data are available minus the `shift_len`-parameter that lets the waveform start a bit before the picks.
